EventsExpress/ClientApp/src/components/helpers/helpers.js

```diff
@@ -49,6 +49,42 @@ export const validate = values => {
         }
     });
 
+    if (!values.inventories || !values.inventories.length) {
```

Comment:
Don't do empty if statements. Invert the check if required.

Reply:
Solved